lambda_functions/vetext_incoming_forwarder_lambda/vetext_incoming_forwarder_lambda.py

```diff
@@ -202,43 +195,36 @@ def make_vetext_request(request_body):
         "body": request_body.get("Body", "")
     }
 
-    json_data = json.dumps(body)
-
-    logger.info(f"Making POST Request to VeText using: ${domain}${path}")
-    logger.debug(f"json dumps: {json_data}")
-
-    connection = None
+    endpoint_uri = f"https://{domain}{path}"
 
-    try:
-        connection = http.client.HTTPSConnection(domain, timeout=6, context=ssl._create_unverified_context())
-        logger.info("generated connection to VeText")
-
-        connection.request(
-            'POST',
-            path,
-            json_data,
-            headers)
-
-        response = connection.getresponse()
-
-        logger.info(f"VeText call complete with response: {response.status}")
-        logger.debug(f"VeText response: {response}")
+    logger.info(f"Making POST Request to VeText using: {endpoint_uri}")
+    logger.debug(f"json dumps: {json.dumps(body)}")
 
-        if response.status == 200:
-            return response
-
-        logger.error("VeText call failed.")
-    except http.client.HTTPException as e:
-        logger.error("HttpException With Call To VeText")
+    try:        
+        response = requests.post(
+            endpoint_uri,
+            verify=False,
+            json=body,
```

Comment:
"body" could be an empty string (line 195).  In that case, do you still want to make a request?

Reply:
yeah, it's possible to be empty but we need to send those to VeText anyway.  Whatever we get from the end-user should be forwarded regardless of content